ci/bump_version.sh

```diff
@@ -31,7 +31,7 @@ bump_version() {
   find_pattern='^version\s*=.*[^,]\s*$'
   replace_pattern='s/'$find_pattern'/version = "'"$version"'"/'
 
-  grep -q "$find_pattern" "$file" && sed -i "$replace_pattern" "$file"
+  grep -q "$find_pattern" "$file" && sed -i "" "$replace_pattern" "$file"
```

Comment:
If you turn out to need different code for different operating systems, there's an example of that in `node/ci/integration_tests.sh`.

Reply:
The code modifications are inside #281